api/core/storage/clickhouse/clickhouse_client.py

```diff
@@ -522,19 +522,23 @@ async def run_count_by_agent_uid(
             )
 
     @override
-    async def list_runs_since(
+    async def list_latest_runs(
         self,
-        since_date: datetime,
+        task_uid: int | None = None,
+        since_date: datetime | None = None,
         is_active: bool = True,
         limit: int = 100,
     ) -> AsyncIterator[AgentRun]:
-        # TODO: use SerializableTaskRunQuery instead
```

Comment:
is that no longer true ? 
I am not 100% sure this function is useful ? Might be good to avoid duplicating logic ?

Reply:
still true